backend/chat/chat-listeners/active-user-handler.js

```diff
@@ -96,13 +96,15 @@ async function updateUserOnlineStatus(userDetails, updateDb = false) {
         onlineUsers.set(userDetails.id, true, ONLINE_TIMEOUT);
 
         const roles = await chatRolesManager.getUsersChatRoles(userDetails.id);
+        const details = await userDatabase.getUserById(userDetails.id);
```

Comment:
There is a new possibly DB hit here to get user settings from the user modal and add it to the data sent to the front end.

Reply:
This is a good callout and actually might be something we can avoid. Both functions that call `updateUserOnlineStatus` (namely `addOnlineUser` and `addActiveUser`) already get user data from the database, so we could instead make `disableViewerList` an argument of `updateUserOnlineStatus` and pass the bool in from those other funcs.

Additionally, a cool little trick to ensure a value is a boolean is to use a double-bang, like so: `!!details.disableViewerList`. Its functionally equivalent to `details.disableViewerList ? details.disableViewerList : false`